client/python/agent_protocol_client/api/agent_api.py

```diff
@@ -1632,7 +1632,7 @@ def upload_agent_task_artifacts_with_http_info(
         }
 
         return self.api_client.call_api(
-            "/agent/tasks/{task_id}/artifacts",
```

Comment:
could we move those version into an env variable? So that future changes can be easier?

Reply:
We likely can, yes. I want to move the client into its own repository as well.